### DISCUSSION TITLE
Can you share the script for the experiment?

### DISCUSSION THREAD

**Discussion by Ningyu-y** [parthsarthi03/raptor#60] [Q&A]
Can you share the script for the experiment? When I wrote my own script test, the results were very different from the data in the paper, and the results were far less good than the data in the paper

**Comment by atirathk** [comment]
@parthsarthi03 bumping this in case it got lost. @Ningyu-y, are you running your scripts on a machine with different specs? If your machine is less powerful than the one that the researchers used, maybe that's why the performance isn't so good?
